mod/projects/set/type_plus_right/project/year.rb

```diff
@@ -37,6 +37,6 @@ def year_progress_table
                    card.all_year_project_cards,
                    [:fancy_year, :research_progress_bar],
                    header: ["Year", "Answers Researched"],
-                   td: { classes: ["year-answer"] }
+                   td: { classes: ["year-answer", "overall-progress-box"] }
```

Comment:
does this mean the year will have the same color as the overall progress bar?

Reply:
`overall-progress-box` has default progress bar colours. Maybe I can  rename the class name with more generic term

Reply:
cool